geoparquet/geoparquet-example.ipynb

```diff
@@ -0,0 +1,237 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %time gdf = gpd.read_parquet(url, columns=["ID_PARCEL", "geometry"], filesystem=filesystem)
</pre><p>Just noting the current output is an error</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>